engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailActorsResolver.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.cloud.notifications.processors.email;
+
+import com.redhat.cloud.notifications.models.Event;
+import com.redhat.cloud.notifications.processors.email.connector.dto.EmailSenderDefaultRecipientDTO;
+import jakarta.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class EmailActorsResolver {
```

Comment:
What's the plan with this? How will the value by dynamically changed based on the app (or bundle)?

Reply:
I didn't know where to put the function that returns the desired sender, so that is why I thought of putting it in a separate class. That way if we decide to use the DB or a static file which is more likely we can simply change the implementation there and call it a day.

Because right now there are two classes that require resolving the sender: the `EmailProcessor` and the `EmailSubscriptionTypeProcessor`